### PR TITLE
Updated codebase from python2 to python3

### PR DESCRIPTION
**Pull request by knekvasil**

### CODE REVIEW

File: Game/Board.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
-from __future__ import unicode_literals
-import History, Pieces, Position
+
+from . import History, Pieces, Position
```
Comment (ritwikd) [inline Game/Board.py:3]: I am unfamiliar with this from . Syntax. Is it functionally equivalent to just import? Perhaps for code clarity we make that change.

File: Game/Board.py [hunk 2]
```diff
@@ -197,11 +197,11 @@ def rook_conflict(self, P_n, turn):
...
 
         # Column
-        Ps_c += map(lambda n: Position.Position(c_c, n), range(8))
+        Ps_c += [Position.Position(c_c, n) for n in range(8)]
         # Row
-        Ps_c += map(lambda n: Position.Position(n, r_c), range(8))
+        Ps_c += [Position.Position(n, r_c) for n in range(8)]
 
-        R_c = map(self.get_piece, Ps_c)
+        R_c = list(map(self.get_piece, Ps_c))
```
Comment (ritwikd) [inline Game/Board.py:204]: Good use of list comprehension.

File: Game/History/__init__.py [hunk 3]
```diff
@@ -1 +1 @@
-import Log, Move
\ No newline at end of file
+from . import Log, Move
```
Comment (ritwikd) [inline Game/History/__init__.py:1]: Same as other comment

**Review by ritwikd — COMMENTED** [review]
Looks good overall! Can you provide a screenshot showing full passage of the unit test suite?

**Comment by knekvasil** [comment]
- Both CheckTest.py and DoMove.py pass:
Screen Shot 2021-03-06 at 4 55 16 PM
Screen Shot 2021-03-06 at 4 54 56 PM

- All the tests in AlgNote.py and ValidMove.py fail because they haven't been worked on yet and have the placeholder: `self.assertEqual(True, False)`
So, expected.

**Comment by ritwikd** [comment]
Neato!

**Pull request merged**